index.html

```diff
@@ -1,39 +1,93 @@
 
-    <!--Import Google Icon Font-->
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <!--Let browser know website is optimized for mobile-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <!-- Compiled and minified CSS -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
-
-
-    <title>Document</title>
+  <!--Import Google Icon Font-->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <!-- Compiled and minified CSS -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
 </head>
 
 <body>
 
+  <!-- test
+  <div class="container">
+    <div class="row">
+      <div class="col s1">1</div>
+      <div class="col s1">2</div>
+      <div class="col s1">3</div>
+      <div class="col s1">4</div>
+      <div class="col s1">5</div>
+      <div class="col s1">6</div>
+      <div class="col s1">7</div>
+      <div class="col s1">8</div>
+      <div class="col s1">9</div>
+      <div class="col s1">10</div>
+      <div class="col s1">11</div>
+      <div class="col s1">12</div>
+    </div>
+  </div>
 
+  <div class="row">
+    <div class="col s1">1</div>
+    <div class="col s1">2</div>
+    <div class="col s1">3</div>
+    <div class="col s1">4</div>
+    <div class="col s1">5</div>
+    <div class="col s1">6</div>
+    <div class="col s1">7</div>
+    <div class="col s1">8</div>
+    <div class="col s1">9</div>
+    <div class="col s1">10</div>
+    <div class="col s1">11</div>
+    <div class="col s1">12</div>
+  </div> -->
 
+  <!--center image-->
+  <div class="container">
+    <div class="row">
+      <div class="col s12 offset-m2">
+        <img class="responsive-img" src="Images/NewsLayout.jpg" height="270" width="571">
+      </div>
+    </div>
+  </div>
 
+  <!--input bar-->
+  <div class="container">
+    <div class="row">
+      <form class="col s12 offset-m3">
+        <div class="row">
+          <div class="input-field col s6">
+            <input placeholder="search" id="search" type="text" class="validate">
+          </div>
+        </div>
+      </form>
+    </div>
+  </div>
 
+  <!--search button-->
+  <div class="container">
+    <div class="row">
+      <a class="col S2 waves-effect waves-light btn offset-m5"><i class="material-icons left">search</i>Search</a>
+    </div>
+  </div>
 
+  <!-- Compiled and minified JavaScript for Materialize -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+  <script src="https://code.jquery.com/jquery-3.4.1.min.js"
+    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
```

Comment:
hey, what do we need popper.js for? is that a part of materializes or is it for something else?

Reply:
Popper.js is required IF you're using Bootstrap JS. It's not required if you're using only the Bootstrap CSS.  So we can remove it.